kahuna/public/js/leases/leases.html

```diff
@@ -0,0 +1,83 @@
+<dt class="metadata-line__key flex-container">
+  <span class="flex-spacer">Leases</span>
+  <button ng:class="{'small': ctrl.grSmall}"
+          ng:click="ctrl.editing = true"
+          gr:tooltip="Add lease"
+          gr:tooltip-position="left">
+
+      <gr-icon ng-show="!ctrl.editing" ng:class="{'spin': ctrl.adding}">add_box</gr-icon>
+      <span>
+         <span ng:show="ctrl.adding">Adding...</span>
+      </span>
+  </button>
+</dt>
+
+<form class="lease__form"
+      ng:show="!ctrl.adding && ctrl.editing"
+      ng:submit="ctrl.save()">
+
+  <select id="access-select" ng:model="ctrl.access" ng-required="required">
+      <option ng-selected="true" value="">Please select access</option>
+      <option value="allow">Allow cropping</option>
+      <option value="deny">Deny cropping</option>
+  </select>
+  <gu-date-range class="lease__date"
+     gu:start-date="ctrl.newLease.startDate"
+     gu:end-date="ctrl.newLease.endDate"
+     gu:show-extras="false"
+     gu:first-day="1">
+  </gu-date-range>
+
+  <input type="text" ng:model="ctrl.newLease.notes" placeholder="Notes..."/>
+
+  <div class="lease__form__buttons">
+    <button class="lease__form__buttons__button-cancel button-cancel" type="button" ng:click="ctrl.cancel()" title="Close">
+        <gr-icon-label gr-icon="close"><span ng:hide="ctrl.grSmall">Cancel</span></gr-icon-label>
+    </button>
+    <button
+        class="lease__form__buttons__button-save button-save"
```

Comment:
I think the class here is getting a bit specific. Are they styles here specifically linked to being part of leases or can this become its own block?